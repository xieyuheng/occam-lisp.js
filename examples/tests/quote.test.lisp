'123
'abc
'"a b c"
'""

(quote 123)
(quote abc)
(quote "a b c")
(quote "")
