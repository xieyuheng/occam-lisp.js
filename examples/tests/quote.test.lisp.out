123
'abc
"a b c"
""
['a]
['a 'b 'c]
[1 2 3]
['a 'b 'c 1 2 3 "a b c"]
123
'abc
"a b c"
""
['a]
['a 'b 'c]
[1 2 3]
['a 'b 'c 1 2 3 "a b c"]
